Failed to build on arm64

Buildlog: https://buildd.debian.org/status/fetch.php?pkg=highwayhash&arch=arm64&ver=0%7Egit20190222.276dd7b-1&stamp=1550977085&raw=0

Version: 0~git20190222.276dd7b (current master)

I have the same issue when cross-compiling HighwayHash to arm64. My log is very much alike to the one from the previous message. The causing error seems to be:

`make HH_AARCH64=1`
.....
`./highwayhash/hh_neon.h:227:76: error: no matching function for call to 'highwayhash::NEON::V128<unsigned int>::ShiftRightInsert(highwayhash::NEON::V2x64U&, long unsigned int)'`

The versions before the commit https://github.com/google/highwayhash/commit/ff5f27d785bda44e1ce0572b7f44943984ced61a work just fine.

--- a/highwayhash/hh_neon.h
+++ b/highwayhash/hh_neon.h
@@ -218,16 +218,6 @@ class HHStateNEON {
     );
   }
 
-  // Rotates 32-bit lanes by "count" bits.
-  static HH_INLINE void Rotate32By(V2x64U* HH_RESTRICT vH,
-                                   V2x64U* HH_RESTRICT vL,
-                                   const uint64_t count) {
-    // WARNING: the shift count is 64 bits, so we can't reuse vsize_mod32,
-    // which is broadcast into 32-bit lanes.
-    *vL = V2x64U((V4x32U(*vL) << int(count)).ShiftRightInsert(*vL, 32-count));
-    *vH = V2x64U((V4x32U(*vH) << int(count)).ShiftRightInsert(*vH, 32-count));
-  }
-
   static HH_INLINE V2x64U ZipperMerge(const V2x64U& v) {
     // Multiplication mixes/scrambles bytes 0-7 of the 64-bit result to
     // varying degrees. In descending order of goodness, bytes

